jenkins-examples/pipeline-examples/declarative-examples/jfrog-distribution-example/Jenkinsfile

```diff
@@ -0,0 +1,114 @@
+pipeline {
+    agent any
+    stages {
+        stage ('Clone') {
+            steps {
+                git branch: 'master', url: "https://github.com/jfrog/jenkins-artifactory-plugin.git"
+            }
+        }
+
+        stage ('Upload file') {
+            steps {
+                rtUpload (
+                    // Obtain an Artifactory server instance, defined in Jenkins --> Manage Jenkins --> Configure System:
+                    serverId: SERVER_ID,
+                    spec: """{
+                            "files": [
+                                    {
+                                        "pattern": "jenkins-examples/pipeline-examples/resources/ArtifactoryPipeline.zip",
+                                        "target": "libs-release-local"
+                                    }
+                                ]
+                            }"""
+                )
+            }
+        }
+
+        stage ('Create release bundle') {
+            steps {
+                dsCreateReleaseBundle(
+                    serverId: SERVER_ID,
+                    name: "example-release-bundle",
+                    version: "1",
+                    spec: """{
+                              "files": [
+                                {
+                                    "pattern": "libs-release-local/ArtifactoryPipeline.zip"
+                                }
+                             ]
+                            }"""
```

Comment:
I think this line should be pushed 1 character forward (additional white-space before }""" so that it is aligned with the openning of the file spec.
The other specs in this examples shoudl also be aligned.
Same goes for the scripted example.